src/components/ViewCard.svelte

```diff
@@ -133,13 +133,13 @@
     </ModalBody>
     <ModalFooter>
       <Container>
-        <Row>
-          <Col xs="4">
+        <Row class="view-card-container">
+          <Col class="view-card-left-half" xs="4.5">
             {#if !card.is_archived}
               <ColorPicker bind:color={card.color} />
             {/if}
           </Col>
-          <Col xs="8">
+          <Col class="view-card-right-half" xs="7.5">
             <ArchiveCard bind:is_archived={card.is_archived} />
```

Comment:
I don't know if decimal values passed to the `xs` prop are allowed (lines 137 and 142). Can we convert them into integers to make it more clear how it is divided along Bootstrap's 12-wide layout?

Reply:
The use of decimal values is what gave results for the button placement.